### DISCUSSION TITLE
support cdx?

### DISCUSSION THREAD

**Discussion by pa-nana** [epam/ketcher#1605] [Q&A] [answered]
Dear Ketcher team,

Thanks for the excellent Ketcher editor :thumbsup:
I found this [https://github.com/epam/ketcher/issues/920](https://github.com/epam/ketcher/issues/920) so the cdxml file format is supported. Then, I would like to know, do you have a plan to support cdx file format?

**Comment by AndreiMazol** [comment, marked as answer]
Dear @CLLPCH ,

your request has been discussed internally and we have decided to put it into our backlog. it will be implemented in scope of https://github.com/epam/ketcher/issues/1640

Regards,
Ketcher team

**Reply by pa-nana** [reply]
Hi @AndreiMazol ,

Thanks a lot!
